.github/workflows/cli-tests.yaml

```diff
@@ -7,11 +7,6 @@ concurrency:
 
 on:
   pull_request:
-  push:
-    branches-ignore:
-      - main
-    paths-ignore:
-      - 'docs/**'
```

Comment:
We don't want to ignore `docs/`?

Reply:
This ignore was only being applied to `push` and nog `pull_request`. Did you mean to ignore for both?